### ISSUE TITLE
[Question] Column identifier from aliased table

### ISSUE THREAD

**Issue by deitrix** [doug-martin/goqu#203]
Hi Doug, quick question regarding table aliases. I'm not sure if this is something already built into the API, or if I'm doing it wrong.

I can't seem to figure out a way to generate the following query:

```mysql
SELECT u.id, u.name, u.created_at, u.updated_at
FROM company_auth.users AS u
```

I would expect the code to look something along the lines of:

```go
dialect := goqu.Dialect("mysql")

// Schema definitions.
authSchema := goqu.S("company_auth")

// Table definitions
usersTable := authSchema.Table("users")

u := usersTable.As("u")

query := dialect.From(u).Select(
	u.Col("id"),
	u.Col("name"),
	u.Col("created_at"),
	u.Col("updated_at"),
)
```

But, this does not compile.

I've noticed that I can retrieve an `IdentifierExpression` from an `AliasedExpression` by calling `GetAs()`, and while this does compile, it doesn't yield the desired query:

```go
query := dialect.From(u).Select(
	u.GetAs().Col("id"),
	u.GetAs().Col("name"),
	u.GetAs().Col("created_at"),
	u.GetAs().Col("updated_at"),
)
```

generated query:
```mysql
SELECT id, name, created_at, updated_at
FROM company_auth.users AS u
```

**Dialect**
- [ ] postgres
- [x] mysql
- [ ] sqlite3

**Comment by doug-martin** [comment]
Hi @deitrix sorry this has taken me a while to get back to you.

Yeah I'll need to take some time to create a fix for this.

While not ideal a work around would be 

```go
dialect := goqu.Dialect("mysql")

// Schema definitions.
authSchema := goqu.S("company_auth")

// Table definitions
usersTable := authSchema.Table("users")

u := goqu.T("u")

sql, _, _ = dialect.From(usersTable.As("u")).Select(
	u.Col("id"),
	u.Col("name"),
	u.Col("created_at"),
	u.Col("updated_at"),
).ToSQL()
fmt.Println(sql)
```

Output:

```
SELECT `u`.`id`, `u`.`name`, `u`.`created_at`, `u`.`updated_at` FROM `company_auth`.`users` AS `u`
```

**Comment by doug-martin** [comment]
Released in `v9.8.0`
